plat/allwinner/common/sunxi_pm.c

```diff
@@ -8,18 +8,47 @@
 #include <assert.h>
 #include <debug.h>
 #include <delay_timer.h>
+#include <gicv2.h>
 #include <mmio.h>
 #include <platform.h>
 #include <platform_def.h>
 #include <psci.h>
 #include <sunxi_mmap.h>
+#include <sunxi_cpucfg.h>
 
 #define SUNXI_WDOG0_CTRL_REG		(SUNXI_WDOG_BASE + 0x0010)
 #define SUNXI_WDOG0_CFG_REG		(SUNXI_WDOG_BASE + 0x0014)
 #define SUNXI_WDOG0_MODE_REG		(SUNXI_WDOG_BASE + 0x0018)
 
 #include "sunxi_private.h"
 
+#define mpidr_is_valid(mpidr) ( \
```

Comment:
This can be simplified to `#define mpidr_is_valid(mpidr) (plat_core_pos_by_mpidr(mpidr) >= 0)`

Reply:
And since the macro is only used once, it can be inlined into the function.